src/publisher/react-devtools-hook.ts

```diff
@@ -47,6 +48,8 @@ export function createReactDevtoolsHook(
     // see https://github.com/facebook/react/blob/4ff5f5719b348d9d8db14aaa49a48532defb4ab7/packages/react-refresh/src/ReactFreshRuntime.js#L509
     renderers,
 
+    rendererInterfaces: new Map<number, ReactIntegration>(),
```

Comment:
We should avoid to add anything to a hook, since it goes to global and used by React and other stuff.